doc/api/process.md

```diff
@@ -1716,7 +1716,7 @@ that started the Node.js process. Symbolic links, if any, are resolved.
 added: v0.1.13
 -->
 
-* `code` {integer} The exit code. **Default:** `0`.
+* `code` {integer} The exit code in the range `0` – `255`. **Default:** `0`.
```

Comment:
The allowed range depends on the operating system. Windows, for example, allows values greater than 255.